CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_hubspot_source v0.12.1
+
+Teeny tiny release!
```

Comment:
My PR is a breaking change so we can just fold that in with this but make it breaking. Something we can handle during the merge